### ISSUE TITLE
Dogs stay frozen from Widows Wine Effect.

### ISSUE THREAD

**Issue by DjNathann** [Zet0rz/nzombies#432]
When throwing/getting hit at/by a dog the dog freezes if you have widows wine.

**Comment by minin43** [comment]
Do you know what Widow's Wine does?

**Comment by DjNathann** [comment]
Yea but the dog stays frozen.

**Comment by minin43** [comment]
Okay so maybe reword the bug.

"Hellhounds remain frozen and unable to move/attack long after the Widow's Wine effect wears away."

**Comment by DjNathann** [comment]
As a Level 413 on Black ops 3 i totally don't know what widows wine does.

**Comment by minin43** [comment]
Yea cause I just know off the top of my head how experienced you are playing BO3...

The way you've worded the problem suggested something different from what you intended. Please confirm my suggested "re-wording" is what you actually intended to say, so we can better help solve the issue.

**Comment by Zet0rz** [comment]
Widows Wine is meant to freeze dogs for 30 seconds. Do they stay frozen after those 30 seconds or did you just think this was a bug when it was just the perk's effect?

**Comment by DjNathann** [comment]
They stay frozen.
